hack/update-codegen-dockerized.sh

```diff
@@ -91,7 +91,7 @@ MOCKGEN_TARGETS=(
   "pkg/ovs/ovsctl OVSCtlClient"
   "pkg/agent/querier AgentQuerier"
   "pkg/controller/querier ControllerQuerier"
-  "pkg/querier AgentNetworkPolicyInfoQuerier"
+  "pkg/querier AgentNetworkPolicyInfoQuerier,ControllerNetworkPolicyInfoQuerier"
```

Comment:
I don't think you need to generate a mock for `ControllerNetworkPolicyInfoQuerier`. If yes, where are you using it?

However, you **do** need to generate the mock for `EndpointQuerier`. I see that you have the generated mock file checked-in, but I don't know how it was generated since `hack/update-codegen-dockerized.sh` does not reference `EndpointQuerier`.